example_package/config.yml

```diff
@@ -74,8 +86,11 @@ scores:
 
 ### expected scores
 
+# sinol_expected_scores: {}
```

Comment:
To chyba nie jest potrzebne, bo 7 linii niżej mamy odkomentowane. Ewentualnie można zostawić tylko zakomentowane, ale idk.

Reply:
st-make sam to będzie chciał dodać więc lepiej zostawić odkomentowane a zakomentowane zostawić bo spójność z tym co wyżej jest

Reply:
No ok, chociaż sinol-make przy automatycznym zmienianiu configa i tak wszystko formatuje po swojemu, m.in. usuwając komentarze. Fajnie